css/styles.css

```diff
@@ -236,9 +236,18 @@ li {
 
   .user-random-button,
   .favorite-button {
+    cursor: pointer;
     padding: 0.8rem;
   }
 
+  .fa-star:hover {
+    transform: scale(1.3);
+  }
+
+  .fa-dice:hover {
+    animation: rotation 0.2s infinite linear;
```

Comment:
You should try `0.02s` for duration, that icon is spinning **really** fast right now 😂